src/components/breadcrumb/index.js

```diff
@@ -1,49 +1,86 @@
 // https://github.com/alphagov/govuk-frontend/blob/master/src/components/breadcrumb/_breadcrumb.scss
 import React from 'react'
-import styled from 'styled-components'
```

Comment:
pretty sure this was merged in to master already, can you merge master in to your branch and push again so the PR reflects your changes?